configuration/blockly/rules-blockly-items-things.md

```diff
@@ -141,7 +141,7 @@ These attributes are returned with the following types:
 - state: State
 - numeric state: Number
 - quantity state: [Quantity](rules-blockly-uom.html#unit-of-measurement-blocks)
-- category: String
+- category:
```

Comment:
Is it correct, that this return type is completely empty?

Reply:
No, good catch. That got lost during some merge issue on my side.